### DISCUSSION TITLE
How do you quickly edit a memo?

### DISCUSSION THREAD

**Discussion by lildata** [usememos/memos#3132] [Q&A]
Hi,

Something I have been struggling with is quickly putting a memo in edit mode.

As far as I see I have to go in the menu and select `Edit`:
![Screenshot from 2024-03-26 11-20-20](https://github.com/usememos/memos/assets/7280019/5772c141-3039-4bcf-b36f-ce982044b5c3)

There are also shortcuts in the top right but only for reaction and comments:
![memos](https://github.com/usememos/memos/assets/7280019/f1f3a23a-f882-4920-8bb4-8e8c2367a51f)

Is there a way to add a Custom Script (https://www.usememos.com/docs/advanced-settings/custom-style-and-script) to make it behave like Google Keep and a click anywhere on the memo to trigger the edit mode?
... or add a shortcut in the top right?

Many thanks

**Comment by owenjuni** [comment]
I also want to know
